testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -577,7 +577,7 @@ Feature: Synchronize products in the products page of the Setup Wizard
     And I select "SUSE Liberty Linux LTSS 7 x86_64" as a product
     Then I should see the "SUSE Liberty Linux LTSS 7 x86_64" selected
     When I open the sub-list of the product "SUSE Liberty Linux LTSS 7 x86_64"
-    And I should see the "SUSE Manager Client Tools for RHEL, Liberty and Clones 7 x86_64" selected
+    And I should see the "SUSE Multi-Linux Manager Client Tools for SUSE Liberty Linux 7, RHEL and clones 7 x86_64 (BETA)" selected
```

Comment:
https://scc.suse.com/admin/products/3033

There's still a typo in SCC :( 
They called it `SUSE Multi-Linux Manager Client Tools for SUSE Liberty Linux 7, RHEL and clones 7 x86_64 (BETA)` but it should be 
`SUSE Multi-Linux Manager Client Tools for SUSE Liberty Linux 7, RHEL and clones x86_64 (BETA)`

your PR is correct for now. I'll check with Abid tomorrow what he wants to do